Traceability interop

mkhraisha opened the pull request:
Adds Traceability Interop spec following procedure in the readme

dontcallmedom reviewed (changes requested):
* the fileame should be index.html rather than `unofficial-…`
* the `CG-FINAL-traceability…` directory should be part of the `credentials` directory (since this is report from the credentials CG)
* the "latest published version" URI is wrong
* conversely, you may want to set a "This version" URI

mkhraisha commented:
@dontcallmedom
Updated filename,
Moved to credentials,
Updated URL for latest version I used https://www.w3.org/community/reports/ccg/CG-FINAL-traceability-interop-20241204 as i believe that is where it will end up but please let me know if im supposed to do anything else

dontcallmedom commented on credentials/CG-FINAL-traceability-interop-20241204/index.html:
```diff
@@ -0,0 +1,1341 @@
+    particularly as they relate to supply chain use cases.
+    Exchanged Verifiable Credentials often implement schemas defined in
+    the Traceability Vocabulary [TRACE-VOCAB]..">
+<link rel="canonical" href="https://www.w3.org/community/reports/ccg/CG-FINAL-traceability-interop-20241204">
```
```suggestion
<link rel="canonical" href="https://www.w3.org/community/reports/credentials/CG-FINAL-traceability-interop-20241204">
```

dontcallmedom commented on credentials/CG-FINAL-traceability-interop-20241204/index.html:
```diff
@@ -0,0 +1,1341 @@
+              <a class="u-url" href="https://www.w3.org/community/reports/credentials/CG-FINAL-traceability-interop-20241204/">https://www.w3.org/community/reports/credentials/CG-FINAL-traceability-interop-20241204/</a>
+            </dd>
+      <dt>Latest published version:</dt><dd>
+              <a href="https://www.w3.org/community/reports/ccg/CG-FINAL-traceability-interop-20241204">https://www.w3.org/community/reports/ccg/CG-FINAL-traceability-interop-20241204</a>
```
```suggestion
              <a href="https://www.w3.org/community/reports/credentials/CG-FINAL-traceability-interop-20241204">https://www.w3.org/community/reports/credentials/CG-FINAL-traceability-interop-20241204</a>
```
